README.md

```diff
@@ -207,9 +207,59 @@ want.
   ]
 }
 ```
+<br />
+
+#### Image dataset format
+- Image Datasets have 2 components:
+  - **Set of images.**
+  - **Labels file** - A mapping from each image to a label. This mapping can be supplied either in a .csv, .xls/.xlsx, or json format.
 
-</details>
-</details>
+###### Labels file format
+- must have an **ID column** (`vizzy_id` in the example below) - a column containing identifiers that uniquely identify each row.
+- must have a **filepath column** (`vizzy_path` in the example below) that contains path to the image file.
```

Comment:
Clarify what are constraints (if any) on where the files must be located. Otherwise clarify these files can live anywhere as long as their full paths are listed in the labels file